src/main/java/rlbot/cppinterop/RLBotDll.java

```diff
@@ -39,6 +39,7 @@
     private static native ByteBufferStruct GetBallPrediction();
     private static native int SendQuickChat(Pointer ptr, int size);
     private static native int StartMatchFlatbuffer(Pointer ptr, int size);
+    private static native boolean IsInitialized();
 
     private static boolean isInitialized = false;
```

Comment:
Is this still needed? Feels like it could lead to typo errors.
If you do decide to keep both, please document the difference.

Reply:
Yep, I still want both. Renamed and documented.